### PR TITLE
Additional models to production

### PR DESCRIPTION
**Pull request by mohitseth-nitrr**

### CODE REVIEW

**Comment by mohitseth-nitrr** [comment]
@ShashankBlr  @vivekjhauipath  lets merge it post production release so that these models are available to everyone.

File: gpu/install_gpu_drivers.sh [hunk 1]
```diff
@@ -87,23 +96,23 @@ function validate_gpu_updated() {
     done
 
     if [ $count == 50 ]; then
-        echo "################## Failed to install gpu ####################"
+        echo "################## Failed to integrate with gpu ####################"
```
Comment (vivekjhauipath) [inline gpu/install_gpu_drivers.sh:99]: nit: shouldn't text match the above

**Review by vivekjhauipath — APPROVED** [review]
LGTM.
@mohitseth-nitrr are we skipping some packages here? Why is hwOCR etc not present?

File: gpu/install_gpu_drivers.sh (hunk 1 above)
Comment (mohitseth-nitrr) [reply gpu/install_gpu_drivers.sh:99]: actually this part is more about verifying integration ... rather than driver installation

**Comment by mohitseth-nitrr** [comment]
@vivekjhauipath  this is complete diff between master and alpha ... did we push HwOCR metadata to alpha ?

**Review by vpn87 — APPROVED** [review]

**Pull request merged**